### PR TITLE
Updates to README.md

### LINKED ISSUE #2: Update README.md

### PR DESCRIPTION
**Pull request by iamMoid**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,33 @@
...
+
+## Functions
+
+The four data visualization functions included in the package are outlined below:
+
+### Boxplot
+
+Returns a boxplot based on the data frame, a numerical feature to view the distribution of and a categorical feature to bucket data into categories. Additionally, there is a boolean option to facet the boxplots into separate charts.
+
+### Correlation Plot
```
Comment (miyer26) [inline README.md:13]: Should we add the function call with the arguments under the header?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,33 @@
...
+
+### Scatterplot
+
+Returns a scatterplot based on the data frame and two numerical feature names passed as the required inputs. An auxiliary input provides the flexibility to color-code the data points based on a categorical feature.
+
+### Fit within the Python ecosystem
+
+Our package will build onto the existing features of 'Altair' using the magma color scheme. It serves as an automated plotter and is a higher level implementation of it. Essentially it circumvents the need to code every single detail and allows the user to focus on the output. We came across two packages that have a similar line of thought:
+
+https://pypi.org/project/deneb/ (Altair) - uses the same base as this package
```
Comment (miyer26) [inline README.md]: Should we have these as hyperlinks?

Comment (miyer26) [reply README.md]: I reckon that may be more useful.

**Comment by miyer26** [comment]
Please find my suggestions above. Thanks!

**Comment by ruben1dlg** [comment]
Other than @miyer26 's observations, I do not have anything else to add 😀

File: README.md (hunk 1 above)
Comment (iamMoid) [reply README.md:13]: I think we can leave as-is for now. I believe the function calls can go under USAGE once we have them ready.

File: README.md (hunk 2 above)
Comment (iamMoid) [reply README.md]: Makes sense. Adding hyperlinks with package names.

File: README.md (hunk 1 above)
Comment (miyer26) [reply README.md:13]: Makes sense, I agree that this is an overview of the functions in the  package

**Review by miyer26 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (shyan0903) [reply README.md]: Could you please update the histogram function to say "Returns a histogram based on the data frame and a categorical feature to plot on the x-axis. The y-axis will display the user-specified aggregation function."

**Review by shyan0903 — APPROVED** [review]

**Review by ruben1dlg — APPROVED** [review]

**Pull request merged**